docs/07_how-to/09_fdc3.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'How to set up interoperability'
+sidebar_label: 'Interoperability/FDC3'
+id: ht-fdc3
+keywords: [fdc3, interoperability]
+tags:
+    - fdc3
+    - interoperability
+
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A Genesis app can use the FDC3 standard for interoperability between front-end applications. 
+
+So, if you have apps that provide services such as charting or news, then you can set up your app so that can share messages with those apps at the front end.
```

Comment:
"so that it"? 

Although I think it needs a slight re-write. Should we say "so you can share the context of your app with another" as opposed to saying about sharing messages? The whole point is one app knows user is looking at something common on another app and updating its context accordingly. Messages are just how it goes about it

Reply:
I don't love the phrase 'share the context of your app' but if it means what it needs mean I'm happy to be out voted.